tests/ballerina-test-utils/build.gradle

```diff
@@ -49,6 +49,10 @@ jar {
     }
 }
 
+publishMavenJavaPublicationToMavenLocal {
+    enabled = false
```

Comment:
Do we need to enable it later? Since it's related to test tasks, we don't need to get it publish to `MavenLocal` right?

Reply:
Since jar task is disabled we need to add this new config to fix build issue. We will not enable later but default it will be true which causes build issue with latest gradle.